domains/afrtite.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "5enox",
+        "email": "animikantan@gmail.com"
+    },
+    "record": {
+        "CNAME": "my-proto.onrender.com"
```

Comment:
Unable to access CNAME.

Reply:
fixed it just now

Reply:
I'm using render.com to host my flask site for now because it's free